feature-libs/my-account/organization/components/shared/organization-table/toggle-link/toggle-link-cell.component.ts

```diff
@@ -0,0 +1,62 @@
+import { ChangeDetectionStrategy, Component, HostBinding } from '@angular/core';
+import {
+  OutletContextData,
+  TableDataOutletContext,
+} from '@spartacus/storefront';
+import { UnitListService } from '../../../unit/services/unit-list.service';
+import { OrganizationCellComponent } from '../organization-cell.component';
+
+@Component({
+  template: `
+    <ng-container *ngIf="count > 0 && nestLevel > 0; else noExpandable">
+      <button class="button action" (click)="toggleItem($event)">
```

Comment:
would be nice to simplify this: 

```html
<button
      *ngIf="count > 0 && nestLevel > 0"
      class="button tree-item-toggle"
      type="button"
      (click)="toggleItem($event)"
    >
      <cx-icon [type]="expanded ? 'CARET_DOWN' : 'CARET_RIGHT'"></cx-icon>
</button>
```

The button type is recommended as technically the button could otherwise submit  a form.

Reply:
I'd use a bit more semantic  class for the button, would also be good to add expand vs collapse as a class name.

Reply:
In the styling we should make the button full height/width of the cell, so that it's easily clickable.